Software/src/main/java/org/foi/emp/carmanagement/services/EmailSenderService.java

```diff
@@ -11,7 +11,7 @@
     @Autowired
     private JavaMailSender javaMailSender;
 
```

Comment:
Apply code format and optimization of imports. 

`CTRL + ALT + L`
`CTRL + ALT + O`